pom.xml

```diff
@@ -23,6 +23,9 @@
   <groupId>co.cask.cdap</groupId>
   <artifactId>cdap</artifactId>
   <version>3.1.0-SNAPSHOT</version>
+  <modules>
+    <module>cdap-formats</module>
```

Comment:
Don't add it here. Add it near the end of this file, under the profile `<id>default</id>`.